Services/AuthenticationService/Commands/LogoutCommand.cs

```diff
@@ -24,7 +24,7 @@ public bool Execute(Guid userId)
             if (userId == Guid.Empty)
             {
                 var e = new BadRequestException("Empty id in log out request.");
-                logger.LogWarning(e, "BadRequest thrown while trying to log out.");
+                logger.LogWarning(e, $"Attempt to logout user with empty id.");

```

Comment:
Why do you need $ here?

Reply:
> Why do you need $ here?

Don't need it here, thanks, will change in next commit